src/main/service/subscriptionService.ts

```diff
@@ -199,7 +204,151 @@ export class SubscriptionService {
     return payload;
   }
 
+  public async configureListTypeForLocationSubscriptions(userId, listType): Promise<boolean> {
+    return await subscriptionRequests.configureListTypeForLocationSubscriptions(
+      this.createListTypeSubscriptionPayload(userId, listType));
+  }
+
+  createListTypeSubscriptionPayload(userId, listType): object {
+    let listTypeArray;
+    if(listType) {
+      if (!Array.isArray(listType)) {
```

Comment:
Can this happen? (should it always be either an array, or blank)

Reply:
I implemented it in the same way function createSubscriptionPayload already there, but I changed it to private now :)